cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -58,6 +58,9 @@ type MasterConfiguration struct {
 	ImageRepository string
 	// UnifiedControlPlaneImage specifies if a specific container image should be used for all control plane components
 	UnifiedControlPlaneImage string
+
+	// FeatureFlags enabled by the user
+	FeatureFlags []string
```

Comment:
I was under the impression that feature flags were stored as `map[string]bool`... 
Is there any easy way for us to do that in any case?
Allows for easy validation